src/UI/Core/Label.h

```diff
@@ -0,0 +1,41 @@
+// ==================================================================================
+// = NAS2D+GUI
+// = Copyright � 2008 - 2012, New Age Software
+// ==================================================================================
+// = This file is part of the NAS2D+GUI library.
+// ==================================================================================
+
+#pragma once
+
+#include "NAS2D/EventHandler.h"
+#include "NAS2D/Timer.h"
+#include "NAS2D/Resources/Image.h"
+#include "Control.h"
+
+#include "NAS2D/Renderer/Color.h"
+#include "NAS2D/Resources/Font.h"
+
+/**
+ * \class TextField
+ * \brief A Basic Text Field.
+ *
+ * The Field Control is just a set of rendered text a user can enter.
```

Comment:
This comment appears to have been copy/pasted and was not updated.

I was wondering why the style was so consistent with existing code until I saw this. :wink: